clients/iceberg-views/src/main/java/org/apache/iceberg/nessie/NessieViewCatalog.java

```diff
@@ -82,7 +82,8 @@ public void initialize(String inputName, Map<String, String> options) {
         x -> x.replace(NessieUtil.NESSIE_CONFIG_PREFIX, "");
 
     this.api =
-        createNessieClientBuilder(options.get(NessieUtil.CONFIG_CLIENT_BUILDER_IMPL))
+        createNessieClientBuilder(
+                options.get(NessieUtil.NESSIE_CONFIG_PREFIX + "client-builder-impl"))
```

Comment:
nit: Should we use the existing `NessieConfigConstants.CONF_NESSIE_CLIENT_BUILDER_IMPL`?